lighthouse-core/report/html/renderer/psi.js

```diff
@@ -64,7 +64,9 @@ function prepareLabData(LHResult, document) {
   scoreGaugeWrapperEl.removeAttribute('href');
 
   const finalScreenshotDataUri = _getFinalScreenshot(perfCategory);
-  return {scoreGaugeEl, perfCategoryEl, finalScreenshotDataUri};
+  const scoreScaleEl = dom.cloneTemplate('#tmpl-lh-scorescale', dom.document()).firstElementChild;
```

Comment:
may not really be a thing, but maybe better to do a `const scoreScaleEl = dom.find('.lh-scorescale', clonedScoreTemplate);`?

More resistant to the unlikely event that an element is added before the `div.lh-scorescale` in the template, but more importantly will throw if the template changes significantly and that class name changes since `dom.find` will throw while `firstElementChild` will happily return whatever is there instead.

Reply:
Yeah good idea.  FirstElementChild was a hack, this seems much safer!